networkx/algorithms/tests/test_matching.py

```diff
@@ -117,14 +117,12 @@ def test_negative_weights(self):
             nx.max_weight_matching(G), matching_dict_to_set({1: 2, 2: 1})
         )
         assert edges_equal(
-            nx.max_weight_matching(G, 1), matching_dict_to_set({1: 3, 2: 4, 3: 1, 4: 2})
+            nx.max_weight_matching(G, maxcardinality=True),
+            matching_dict_to_set({1: 3, 2: 4, 3: 1, 4: 2}),
         )
         assert edges_equal(
             nx.min_weight_matching(G), matching_dict_to_set({1: 2, 3: 4})
         )
-        assert edges_equal(
-            nx.min_weight_matching(G, 1), matching_dict_to_set({1: 2, 3: 4})
```

Comment:
Note in this case, the `maxcardinality` param was passed as a positional argument and also using an `int` instead of `bool` to indicate truthiness - I've updated the test above to make this a bit more clear.